app/serializers/discussion_reader_serializer.rb

```diff
@@ -9,5 +10,13 @@ class DiscussionReaderSerializer < ActiveModel::Serializer
              :read_items_count,
              :read_salient_items_count,
              :last_read_sequence_id,
-             :volume
+             :volume,
+             :is_read
+  def is_read
+    object.last_read_sequence_id == object.discussion.last_sequence_id
+  end
+
+  def group_id
+    object.discussion.group_id
```

Comment:
Biggest thing is that I want to do `collection.find(unread: true)` for discussion readers in a loki query; joining on the client side's too much of a pain I think.

But I can just fiddle this onto the reader on the client side like we do with the discussion_id, probably